Userland/Applications/SystemMonitor/main.cpp

```diff
@@ -385,6 +386,16 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         },
         &process_table_view);
 
+    auto debug_action = GUI::Action::create(
+        "Debug in HackStudio", { Mod_Ctrl, Key_D }, TRY(Gfx::Bitmap::load_from_file("/res/icons/16x16/app-hack-studio.png"sv)), [&](const GUI::Action&) {
+            pid_t pid = selected_id(ProcessModel::Column::PID);
+            if (pid == -1)
```

Comment:
what about pid `0`? can it even show up in the table?

Reply:
Can you clarify? Why would we need special treatment for pid `0`?
(Also, note that the pid retrieval logic here is lifted from other similar context menu items in this table).

Reply:
Oh, I see what you mean.
Well, I also don't see any reason someone would want to attach to the "idle process", but since you can only do that as root I don't see a reason to explicitly disable that option from the `SystemMonitor` GUI.

(I'm resolving this discussion, but feel free to re-open it and comment again ofc.)